seed/static/seed/js/controllers/mapping_controller.js

```diff
@@ -332,22 +386,13 @@ angular.module('BE.seed.controller.mapping', [])
       };
 
       const get_col_from_suggestion = (name) => {
-        if ($scope.import_file.source_type === "BuildingSync Raw") {
-          const suggestion = $scope.suggested_mappings[name];
-
-          return {
-            name: name,
-            suggestion_column_name: suggestion[1],
-            suggestion_table_name: suggestion[0],
-            raw_data: _.map(first_five_rows_payload.first_five_rows, name)
-          };
-        }
```

Comment:
I don't _think_ this is required anymore...

Reply:
I think so too.